### PR TITLE
style: update metric modal peek styles

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: N/A

### Description:

- Some spacing updates, borders, sizes etc
- Added default props to `Paper`, which update the border, shadow used, etc
- Removed usage of `AspectRatio`

Screenshot

Screenshot 2024-11-28 at 15 05 47

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx [hunk 1]
```diff
@@ -94,53 +94,82 @@ export const MetricPeekModal: FC<Props> = ({ opened, onClose }) => {
...
-                    <Flex align="stretch" gap={0}>
-                        <Stack p="md" bg="gray.0" w={360} spacing="sm">
+                <Modal.Body
+                    p={0}
+                    h="100%"
+                    sx={{ display: 'flex' }}
+                    miw={800}
+                    mih={600}
+                >
+                    <Stack p="xl" bg="#FDFDFD" h="100%" w={360} spacing="xl">
```
Comment (IrakliJani) [inline packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx:139]: this color, should we put it somewhere nice?

File: packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx [hunk 2]
```diff
@@ -94,53 +94,82 @@ export const MetricPeekModal: FC<Props> = ({ opened, onClose }) => {
...
                                     size="xs"
-                                    style={{
-                                        visibility: comparisonType
-                                            ? 'visible'
-                                            : 'hidden',
+                                    radius="md"
+                                    sx={{
+                                        visibility:
+                                            comparisonType ===
+                                            MetricExplorerComparison.NONE
```
Comment (IrakliJani) [inline packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx:170]: oh good catch I decided to add `NONE` in the very end and missted it!

File: packages/frontend/src/pages/MetricsCatalog.tsx [hunk 3]
```diff
@@ -35,6 +35,18 @@ const MetricsCatalog: FC = () => {
...
+                        Paper: {
+                            defaultProps: {
+                                radius: 'md',
+                                shadow: 'subtle',
+                                withBorder: true,
+                                sx: (theme: MantineTheme) => ({
+                                    '&[data-with-border]': {
+                                        border: `1px solid ${theme.colors.gray[2]}`,
+                                    },
+                                }),
```
Comment (IrakliJani) [inline packages/frontend/src/pages/MetricsCatalog.tsx:47]: oh ok I thought this was a global thing but its only for metrics catalog page. this makes me what can do in future to scale default theming across the app. maybe a problem for another day!

**Review by IrakliJani — APPROVED** [review]

File: packages/frontend/src/pages/MetricsCatalog.tsx (hunk 3 above)
Comment (joaoviana) [reply packages/frontend/src/pages/MetricsCatalog.tsx:47]: 1000%, with this way we can just merge these changes and move them to the main theme - been trying to get a few cases where this happens, specially the borders always being gray.2 😅

File: packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx (hunk 1 above)
Comment (joaoviana) [reply packages/frontend/src/features/metricsCatalog/components/MetricPeekModal.tsx:139]: agree! Will put in separate pr!

**Pull request merged**